### ISSUE TITLE
SSO/Login: VA.gov Health Pages authenticated content review

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#10317]
## Issue Description

Review content and provide to Liz for mockups.

---
## Tasks
- [x] Review content and provide tracked edits
- [x] Share edits with Peggy to ensure content alignment
- [x] Share with Lauren for questions/concerns
- [x] Get confirmation on ability to show facility-level name and dates
- [x] Finalize alert content and hand off to Liz for mockups
- [x] Finalize full page content and hand off to Liz for mockups/Kelson for implementation

## Acceptance Criteria
- [x] Alert content handed off to Liz for mockups
- [x] Full page content handed off to Liz for mockups/Kelson for implementation

## Documentation

Desktop Invision: https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/409123884_Unauthenticated_Pages

Mobile Invision: https://adhoc.invisionapp.com/share/5QWAJJQNG2J#/409260607_M-Authenticated_Pages_-_Alerts

Usability testing summary: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/sso/ux-research/alerts/research-summary.md

Usability discussion guide: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/sso/ux-research/alerts/discussion-guide.md

Product Outline: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/sso

Handoff document: https://docs.google.com/document/d/11OGKfLk0fKd08pqCKT0Wvrt9ze2vZW_PSEn2kJdK714/edit#

---
## Complexities or points of consideration

There may be some complexity to manage in the future when new Cerner facilities roll out (ex: what happens if a user is a patient at multiple facilities - how would that impact the alert box?)

**Comment by kelsonic** [comment]
Hey @DanielleThierryUSDSVA! Have a quick question:

For [this design spec](https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/404880626) and this CTA widget:

![image](https://user-images.githubusercontent.com/12773166/85044841-6d249900-b14b-11ea-9fd7-0351e4975549.png)

It only mentions 1 cerner facility. However, a user can belong to more than one Cerner facility. Whether the user has 1 or more cerner facilities, the CTA buttons would stay the same I'd imagine but should the content be modified to support more than one? 🙂

**Comment by kelsonic** [comment]
Also, a lot of the auth content design spec doesn't seem dependent on whether or not the user belongs to a Cerner facility. As a developer looking at the design specs, it's not super clear to me when I should show content based on the following user states:

1. User is unauthenticated.
2. User is authenticated and belongs to 0 Cerner facilities.
3. User is authenticated and belongs to 1 or more Cerner facilities and no other facilities (user belongs to only Cerner facilities).
3. User is authenticated and belongs to 1 or more Cerner facilities AND non-Cerner facilities.

**Comment by kelsonic** [comment]
![image](https://user-images.githubusercontent.com/12773166/85143898-986cbe00-b207-11ea-830a-20f687243d99.png)

In the content we see this several times. Does anyone know what the URL should be for that link? 🙂

**Comment by DanielleThierryUSDSVA** [comment]
Hi @kelsonic. I've blocked off part of today to dig into the full content review for this. Let me know if you have time to sync on some of the questions above. Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @kelsonic @peggygannon @andaleliz. After reviewing the knowledge sharing document and the designs/content, here are my initial thoughts for the authenticated alert for users registered at 1 or more Cerner facilities.

**Goals:**

- Keep alert headline as short as possible
- Accommodate multiple potential Cerner health facilities
- Keep content as clear and concise as possible throughout alert
- Align with VA.gov voice, tone, and style guide

I'd love to sync and talk through these internally before proposing to SSO team.

# Option 1

## Send or receive a secure message

_[alert]_

### Our records show you’re registered at a VA health facility that’s using the new My VA Health portal.

**_[Cerner health facility 1]_** is using My VA Health as of **[date]**.
**_[Cerner health facility 2]_** is using VA My VA Health as of **[date]**.

Please choose a health portal below, depending on the providers' facility. You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the same account you used to sign in on VA.gov. 

**Send a secure message to:**

A _[Cerner health facility 1]_ _[or Cerner health facility 2]_ provider

[button] Use My VA Health Secure Messaging

A provider at another VA medical center

[button] Use My HealtheVet Secure Messaging


# Option 2

## Send or receive a secure message

_[alert]_

### Your VA health facility may be using the new My VA Health portal.

**Our records show you're registered at:**
**_[Cerner health facility 1]_** (Using My VA Health as of **_[date]_**)
**_[Cerner health facility 2]_** (Using My VA Health as of **_[date]_**)

Please choose a health portal below, depending on the providers' facility. You may need to disable your browser’s pop-up blocker to open the portal. If you’re prompted to sign in again, use the same account you used to sign in to VA.gov. 

**Send a secure message to:**

A _[Cerner health facility 1]_ _[or Cerner health facility 2]_ provider

[button] Use My VA Health Secure Messaging

A provider at another VA medical center

[button] Use My HealtheVet Secure Messaging

FYSA  @stephorkandatx

**Comment by DanielleThierryUSDSVA** [comment]
**Update:** Peggy and I connected on this today. Option 2 is our top recommendation, as it keeps the header brief and more personalized, but brings in the "you're registered at:" language that's key for connecting Veterans to the idea of where they're registered.

In the eligibility info on the page mockups, we reference being registered at a "VA health facility." In the alert mockups, we reference being registered at a "VA medical center." We will follow up with SSO team to confirm which way to go.

**Comment by kelsonic** [comment]
Thank you @DanielleThierryUSDSVA! I'm not sure who's maintaining the design but could we update it with Option 2? 🙂 

> In the eligibility info on the page mockups, we reference being registered at a "VA health facility." In the alert mockups, we reference being registered at a "VA medical center." We will follow up with SSO team to confirm which way to go.

Sounds great, lmk! 🚀

**Comment by andaleliz** [comment]
@stephorkandatx Am I responsible for making content updates to the designs? I believe they were created in Figma, which is a tool I'm not familiar with (and don't have a license for), so updating these designs may be significantly more time consuming than usual. I'm not sure if there is an easy Figma > Sketch translation.

@kelsonic Do you have it coded out based on the current design spec somewhere I can access? It would probably be less time consuming for me to just update the copy for you using browser dev tools and do a screenshot.

**Comment by kelsonic** [comment]
@andaleliz Unfortunately I don't since each page is now a widget and would have to be added via drupal, which I believe would affect all envs 🤔 but maybe someone with access to drupal could add the widgets to a dummy page to test them out!

**Comment by DanielleThierryUSDSVA** [comment]
@kelsonic @andaleliz I have access to Drupal and can help with adding widgets if needed.

**Comment by andaleliz** [comment]
@DanielleThierryUSDSVA Good to know! I'm working on getting the source files, we should have those anyway. I may also be able to create the mockup with the existing page on staging. Stay tuned.

**Comment by andaleliz** [comment]
@DanielleThierryUSDSVA on second thought - is that Option 2 copy final? I wasn't sure from your message yesterday whether or not you need to review with the SSO team before we move forward.  I'd like to wait to make mock-ups until we're sure of the copy.

**Comment by DanielleThierryUSDSVA** [comment]
@andaleliz Great question. I confirmed with SSO team that they don't need to review. But I'm going to share with Lauren today. I'll tag you here when copy is final. Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @RLHecht. Could you review the attached for any errors/omissions before @andaleliz flows into the mock-ups? This document has updated copy for all of the authenticated page alerts. Thank you!

[SSO content review_alerts_6.25.20](https://app.zenhub.com/files/133843125/512f1c4c-a630-47c9-bede-979c3f36b465/download)

(also note that the dates in here are FPO and will change once we know actual launch dates)

**Comment by RLHecht** [comment]
@DanielleThierryUSDSVA @andaleliz Here's the doc with my edits. Let me know if you need anything else. 
[SSO content review_alerts_6.25.20-RH-edit.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4837405/SSO.content.review_alerts_6.25.20-RH-edit.docx)

**Comment by DanielleThierryUSDSVA** [comment]
Thank you, @RLHecht! Clean version attached. Will wait until Monday to confirm if we need to remove date or make other changes based on availability of data.

[SSO.content.review_alerts_6.25.20_ready for design](https://app.zenhub.com/files/133843125/deb7ac09-4303-496e-a3c4-ad2c7ad70bdc/download)

**Comment by DanielleThierryUSDSVA** [comment]
@peggygannon Here are my recommended edits for the SSO authenticated content. Edits to design versions tracked and in blue. Please let me know any additional changes, etc. before I send to Lauren for final look. Thanks!

[SSO content review_authenticated content_6.30.20](https://app.zenhub.com/files/133843125/7bc8d46c-872f-49be-97db-5b8b9e6890d5/download)

**Comment by DanielleThierryUSDSVA** [comment]
**PM note:** I have changed this issue to reflect review of all _authenticated_ content, ending at passing along to Peggy for alignment. This is because:

a) This is a large volume of content (20 pages + alerts just for authenticated) with no tracked documents to work from (ie, needed to build doc then find what had changed in designs [not all marked] and edit).
b) This isn't launching until August (for 1 page) and September/October for other pages. I need to ensure that I'm prioritizing content reviews effectively based on launch dates.

I'll open new issues for the unauthenticated content review.

(FYSA @kelsonic)

**Comment by peggygannon** [comment]
hi, @DanielleThierryUSDSVA The content looks good. I have no additional edits. 

On Schedule VA appointments,  saw this bullet was removed "Look up past appointments from the last 2 years" Should we update the live page right now with this change too?

**Comment by MickinSahni** [comment]
@DanielleThierryUSDSVA does this mean we can close this out.

#10618 is blocked by this one.

**Comment by DanielleThierryUSDSVA** [comment]
@MickinSahni Please leave open for now, thank you!

**Adding my note from slack here as well:**

We de-prioritized content a bit to focus on getting the links up for testing since this isn't launching until August (for 1 page) and September/October (for the others). Will update when content is ready to flow into design.

**Comment by DanielleThierryUSDSVA** [comment]
@peggygannon:

"Schedule VA appointments, saw this bullet was removed "Look up past appointments from the last 2 years" Should we update the live page right now with this change too?"

- I don't know why this was removed. It's a question I'm sending to Cerner and will follow up. Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
@MickinSahni @andaleliz Sorry for the confusion on this issue! I realized that I'd meant to open a new issue for Lauren's review of authenticated content/confirming if we have the data to show dates, but must have only opened a new one for unauthenticated. I think, though, we can just leave this one open instead for authenticated and I've added the additional steps above. Thanks!

**Comment by MickinSahni** [comment]
> @MickinSahni @andaleliz Sorry for the confusion on this issue! I realized that I'd meant to open a new issue for Lauren's review of authenticated content/confirming if we have the data to show dates, but must have only opened a new one for unauthenticated. I think, though, we can just leave this one open instead for authenticated and I've added the additional steps above. Thanks!

Ah ok. My question was mainly around does it even make sense to add it to this current sprint since it's on-going

**Comment by DanielleThierryUSDSVA** [comment]
Hi @andaleliz. Here's the updated content for the authenticated alerts: [SSO content review_authenticated content_6.30.20](https://app.zenhub.com/files/133843125/4ea0abc9-3547-4527-a710-9464dcbd9252/download)

**Comment by andaleliz** [comment]
Thanks Danielle! Often times when I receive content docs there's heading and styling "mark up" - do you have opinions on that, or prefer to leave to my discretion? [I dropped 2 versions of the first alert into InVision for more clarity.](https://vsateams.invisionapp.com/share/4XY0MRMVE56)  The first is my direct interpretation of your doc, and the second has some additional heading/bold text that might make things more scannable for folks who don't read blocks of text.

Please let me know your thoughts and I'll complete the rest of the screens.  Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @andaleliz. Thanks for checking! Yes, we definitely want to break up copy with bold text. That's in the Word doc (some text is bolded) so I'm not sure why it didn't come through! But glad you checked.

A few thoughts:

- In the doc, the lead-ins (ie, "Our records show you're registered at:" and "Send a secure message to a provider at:") are also bold. I see you've left them unbolded. I do think they anchor the 2 sections (what records show/how to send a message) so I wonder if they should be bold or even subheaders (thinking of screen readers when thinking maybe headers)? Also realize that would then make for a lot of bolding which loses the point of it...just some additional thoughts for when you get back. 

- In the doc, in the records section, just "Mann-Grandstaff VA Medical Center" is bold, and then "(Now using My VA Health)" is not. Could we see how that looks in the mockup?

- I'd just gone with bold above each button. But it looks like you used headers there. I like the headers - let's go with that.

**Comment by andaleliz** [comment]
@DanielleThierryUSDSVA Thanks, I'm glad I checked too! I don't have Word, so maybe it's a Word > Pages issue?

Could you possibly send me a screenshot of how one section looks in your doc, to ensure I'm clear on what you're describing?  Thank you. :)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @andaleliz. Sure, no problem! I'd actually created a quick non-designer "mock up" of sorts for when I was sharing this content with Lauren so she could see how it might look. So I'm adding that below for clarity. Also, just confirmed 2 minor things as part of a review of the full authenticated content so adding version with those highlighted. Thanks!

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/4528b6b4-d276-425b-9957-bfff85966b9b)



[SSO.content.review_alerts_7.10.20_ready for design_no dates](https://app.zenhub.com/files/133843125/bbbe5b69-7da0-4777-b5d2-0a39ff1c2592/download)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @MickinSahni. I've handed off the alert content to Liz, but the full content will need to carry over to next sprint:

- Got Lauren's feedback. One of the pages in the designs (where we picked up the content to review) had outdated content. So I need to do some additional work to update that.
- Also going to connect with Mikki, Nick, and Kelson re: whether we can rethink approach a bit to make it so that we only have to have some content in the react widget and can leave the majority on the static page.

Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @kelsonic. Question for you. I just checked in with Peggy again on the schedule appointments page about why help desk hours are listed in CT (instead of our standard ET). She said it's because they're localized to the user's facility location (which in this case in Ohio), and that the design intent was for the hours to be dynamic based on the user's facility location/timezone as we continue to add facilities in the future. 

Wanted to check in with you to ask whether you coded that based on design specs and, if not, what the level of efforts/potential complexities are for doing that? Happy to hop on a call if it's easier to chat. Thanks!

**Comment by kelsonic** [comment]
@DanielleThierryUSDSVA It's certainly feasible to do CT instead of ET, but I think this should be run by design before we commit to that since other places on the site seem to have a different standard.

**Comment by DanielleThierryUSDSVA** [comment]
Agree 100% - thank you, @kelsonic! I'd marked it as an error, but then Peggy had noted the intent.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @andaleliz. I was just invited to a meeting with My HealtheVet tomorrow morning at 9 a.m. that includes going over pathways/alert content for the Chalmers site (only applies to the schedule/view appointments right now). Would you be able to update the alert mockup for just that page today so I can share ahead of tomorrow morning's meeting? Believe it's on pg. 6 of the Word doc.

**Comment by andaleliz** [comment]
@DanielleThierryUSDSVA [Here you go](https://vsateams.invisionapp.com/share/6DY2YWOE294). If you need any edits, I'll be online until ~ 4pm EST today, then have to hang with Isabel (daughter).

**Comment by andaleliz** [comment]
I went ahead and updated the alerts on all the pages in the content doc - all available at the same link: https://vsateams.invisionapp.com/share/4XY0MRMVE56

**Comment by DanielleThierryUSDSVA** [comment]
Thank you so much, @andaleliz!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @RLHecht. Could you copy edit this doc before I hand it off for implementation? It's similar to the unauthenticated version you edited earlier, but this is the authenticated content. Thank you! 

[SSO_authenticated content_for RH copy edit_7.28.20](https://app.zenhub.com/files/133843125/f919cfc6-9d98-4034-8fdd-1b55e8db3181/download)

**Comment by RLHecht** [comment]
@DanielleThierryUSDSVA Here you go...
[SSO_authenticated content_for RH copy edit_7.28.20-RH-edit.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4988571/SSO_authenticated.content_for.RH.copy.edit_7.28.20-RH-edit.docx)

**Comment by DanielleThierryUSDSVA** [comment]
Uploading final here as well as in implementation ticket.
[https://app.zenhub.com/files/133843125/c540ed80-2e43-42bf-ac92-63f8b71e8c79/download](https://app.zenhub.com/files/133843125/c540ed80-2e43-42bf-ac92-63f8b71e8c79/download)

**Comment by andaleliz** [comment]
Hi, just wanted to double check since I've been out - is anything needed from me here? I don't have any open tickets related to this assigned to me, and I _think_ @kelsonic said he didn't need mock-ups if it was just content changes.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @andaleliz. Thanks for checking! I think you're correct that we're okay without mockups since it's just content changes. But will definitely defer to @kelsonic on confirming that.

Also, @kelsonic, can we connect today or tomorrow on any questions you might have re: the content for the schedule appointments page? I'm going to be out next week (Monday, 8/10 through Wednesday, 8/12) so I just want to make sure to answer any questions well ahead of that so I don't block you. Thanks!
